.github/workflows/protochecker.yml

```diff
@@ -0,0 +1,28 @@
+name: Protobuf Checker
```

Comment:
Care to add `workflow_dispatch` usage here, so it might be triggered manually on any branch (not upon PRs only)?